pkg/annotations/annotations.go

```diff
@@ -612,6 +615,15 @@ func (a *Annotations) setTags(annotations map[string]string) error {
 	return nil
 }
 
+func (a *Annotations) setIgnoreHostHeader(annotations map[string]string) error {
+	if annotations[ignoreHostHeader] == "" {
```

Comment:
Can we use a ParseBool here rather than defaulting to true if its not an empty string?

https://golang.org/pkg/strconv/#ParseBool

Reply:
Sure, done 8f5f2675f4aae64cb73ade76cba514bdf901efda